### ISSUE TITLE
Entries keep disappearing from site

### ISSUE THREAD

**Issue by schmidtkvn** [geoffhumphrey/brewcompetitiononlineentry#488]
I've had three different participants report that they entered beer, then later looked at the site and didn't see them anymore. In two cases they say they entered the beers several times and they kept being deleted. In one case I did notice that the competitor had 3 beers registered one day, then only 1 the second. Whats more, I have 10 participants who paid more than the correct amount for their number of entires, so I'm guessing it has happened to more than just the 3 who reported it. And I only have 42 participants total, so this is almost a quarter of my people!

The competition is this week, so I'm just going to double check everything manually based on the beers that are dropped off. 
I am using 1.3.1.0 configured with the new 2015 categories.

**Comment by GraintoGlass** [comment]
It seems that there is a default to purge entries if they aren't indicated as paid and received every 24 hours.  I discovered that you can change the "Preferences" settings so it won't do that (after going through several rounds of deletions beyond our control). I was able to manually add entries for participants--with new assigned numbers.  This was after the cutoff for our competition entries--and was quite a headache!

**Comment by schmidtkvn** [comment]
I don't think that is just it. On Tuesday of this week I manually sorted all the bottles and had 12 entries with printed bottle slips and entry#, but no corresponding record in the system. Most of these people had paid, all of them had other entries that weren't deleted. This took me up to 107 entries, over my entry limit for the competition but I had enough judges so I decided to just go with it. I then manually entered all the extra entries back into the system so I could get a judging number. Today when I logged back in to print those extra labels I'm back down to 95 entries! I already have the beer and the payment so I'm sure people aren't withdrawing them.

This makes the software worse than useless. I'm having to do the same work over and over again.
